### PR TITLE
small cleanup

### PR DESCRIPTION
**Pull request by ordian**
* addresses https://github.com/paritytech/polkadot/pull/2235#discussion_r555940419
* a follow up to #2262

### CODE REVIEW

**Review by drahnr — APPROVED** [review]
This should be made aware in the appropriate channels to avoid confusion about missing metrics. Other than that :+1:

File: node/overseer/src/lib.rs [hunk 1]
```diff
@@ -1130,26 +1130,34 @@ impl metrics::Metrics for Metrics {
...
 				prometheus::Histogram::with_opts(
 					prometheus::HistogramOpts {
 						common_opts: prometheus::Opts::new(
-							"overseer_channel_fill_level_from_overseer",
+							"parachain_from_overseer_channel_queue_sizes",
 							"Number of elements sitting in the channel waiting to be processed.",
 						),
-						buckets: prometheus::exponential_buckets(0.00001_f64, 2_f64, (CHANNEL_CAPACITY as f64).log2().ceil() as usize).expect("inputs are within documented range; qed"),
+						buckets: prometheus::exponential_buckets(
+							0.00001_f64,
```
Comment (coriolinus) [inline node/overseer/src/lib.rs:1141]: I realize this is just a formatting, but I just realized: channel fill size is an integer, so any buckets between 0 and 1 are wasted.

```suggestion
							1_f64,
```

File: node/overseer/src/lib.rs [hunk 2]
```diff
@@ -1130,26 +1130,34 @@ impl metrics::Metrics for Metrics {
...
 				prometheus::Histogram::with_opts(
 					prometheus::HistogramOpts {
 						common_opts: prometheus::Opts::new(
-							"overseer_channel_fill_level_to_overseer",
+							"parachain_to_overseer_channel_queue_sizes",
 							"Number of elements sitting in the channel waiting to be processed.",
 						),
-						buckets: prometheus::exponential_buckets(0.00001_f64, 2_f64, (CHANNEL_CAPACITY as f64).log2().ceil() as usize).expect("inputs are within documented range; qed"),
+						buckets: prometheus::exponential_buckets(
+							0.00001_f64,
```
Comment (coriolinus) [inline node/overseer/src/lib.rs:1157]: same

```suggestion
							1_f64,
```

**Review by coriolinus — APPROVED** [review]
Some nits not directly related to this cleanup. Cleanup itself looks fine.

**Comment by ordian** [comment]
bot merge

**Pull request merged**